WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -265,7 +265,12 @@ public void onClick(View v) {
         mBtnSpamComment.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                moderateComment(CommentStatus.SPAM);
+                if (mBtnSpamComment.getText().equals(
```

Comment:
I think that's more readable and maintain the code consistency. 
You right, IIRC unspam action should change the status unapproved/hold instead of moving to approved right away.

Let me change these two.